static/js/i18n/place.json

```diff
@@ -89,27 +89,27 @@
   },
   "plural_places": {
     "defaultMessage": "Places",
-    "description": "General collection of places. Used for comparison charts of Places in California."
+    "description": "General collection of places. It is used in several places when we don't have a specific place type. First use case example: for Russia, we simply have a heading Places in Russia on the sidebar with links to many places contained in Russia. Second use case example: for comparison charts, such as Median Age: Places near Sai Kung. Or Median Age: Other Places. Third use case example: for the ranking pages, we may say that the page ranks the life expectancy for All Places in Russia."
   },
   "plural_zip_codes": {
     "defaultMessage": "Zip Codes",
-    "description": "A collection of Zip Codes. Used in ranking pages, etc. E.g. Rankings of Number of Employed People for Zip Codes in USA."
+    "description": "A collection of Zip Codes. It is used in several places when we don't have a specific place type. First use case example: for Russia, we simply have a heading Zip Codes in Fremont on the sidebar with links to many zip code areas contained in Fremont. Second use case example: for comparison charts, such as Median Age: Zip Codes near 94539. Or Median Age: Other Zip Codes. Third use case example: for the ranking pages, we may say Rankings of Number of Employed People for Zip Codes in Santa Clara County."
```

Comment:
Isn't zip code a specific place type (looking at the second sentence).

I think the descriptions would be clearer if you used escaped quotes around the string examples (ditto for the rest)